scripts/prepare-new-env-keys.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ "$#" -ne 1 ]; then
+	echo "Please provide the number of initial collators!"
+	exit 1
+fi
+
+NETWORK=${NETWORK:-altair}
```

Comment:
I know its littel, but can we start making Centrifuge the default for all the scripts instead of Altair?